gen_openj9_templates.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
```

Comment:
Please put this into `templates` directory

Reply:
+1 Maybe we could even have `scripts` subdirectory of the `templates` directory to hold this (and the `generate-template-docs.py` one).

Reply:
Done, there were some issues with a different working directory, but I fixed them